RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java

```diff
@@ -993,56 +993,25 @@ private boolean parseSystemPluginOutput(String regfilePath, AbstractFile regAbst
     private void parseBlueToothDevices(AbstractFile regFile, BufferedReader reader) throws FileNotFoundException, IOException {
         List<BlackboardArtifact> bbartifacts = new ArrayList<>();
         String line = reader.readLine();
-        // date format for plugin Tue Jun 23 10:27:54 2020 Z
-        SimpleDateFormat dateFormat = new SimpleDateFormat("EEE MMM d HH:mm:ss yyyy", US);
-        Long bthLastSeen = Long.valueOf(0);
-        Long bthLastConnected = Long.valueOf(0);
         while (!line.contains(SECTION_DIVIDER)) {
             line = reader.readLine();
```

Comment:
Could you add a check after this readLine to break/return if null?

Reply:
Done